test/jobs/TestJobsFeature.cpp

```diff
@@ -259,7 +259,6 @@ class TestJobsFeature : public ::testing::Test
     void SetUp()
     {
         ThingName = Aws::Crt::String("thing-name value");
-        notifier = shared_ptr<MockNotifier>(new MockNotifier());
```

Comment:
How is this change related to DD?

Reply:
I must have accidentally deleted a line while referencing Jobs tests.